api_server.py

```
@@ -503,10 +553,129 @@ async def chat(
+            # Inject session_id into the done event
+            if event.startswith('data: {"type": "done"'):
+                payload = json.loads(event[6:])
+                payload["session_id"] = session_id
+                yield f"data: {json.dumps(payload)}\n\n"
+            else:
+                # Accumulate chunks to persist in session history after streaming
+                if event.startswith('data: {"type": "chunk"'):
+                    try:
+                        full_answer.append(json.loads(event[6:])["text"])
+                    except Exception:
+                        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)